test/test_runhistory/test_runhistory2epm.py

```diff
@@ -338,6 +338,27 @@ def test_get_X_y(self):
         c_sol = np.array([False, False, True, True])
         self.assertTrue(np.all(c == c_sol))
 
+    def test_budget_selection(self):
+        '''
+            adding some rundata and check budget selection
+        '''
+
+        rh2epm = runhistory2epm.RunHistory2EPM4LogCost(num_params=2,
```

Comment:
Why would you use the LogCost transformer here?

Reply:
Any transformer would work, right? But yes, I'll switch it to regular cost